travis.sh

```diff
@@ -3,11 +3,18 @@ set -e #break script on non-zero exitcode from any command
 gem install bundler
 bundle install
 
+case $OSTYPE in darwin*)
+    brew update > /dev/null
+    brew upgrade cmake
```

Comment:
I guess the people from Travis upgraded the default CMake in their images, because I saw either 3.6.x or 3.7.x in the logs when I bothered to look a while back.